src/commands/dev/lib/dev-builder.ts

```diff
@@ -106,18 +102,20 @@ export async function executeBuild(
   let result: BuildResult;
   try {
     devServer.applyBuildEnv(nowJson);
-    let result = await builder.build({
+    const r = await builder.build({
       files,
       entrypoint,
       workPath,
       config,
       meta: { isDev: true, requestPath }
     });
-    if (!result.output) {
-      // `BuilderOutputs` map was returned
-      result = { output: result as BuilderOutputs };
+    if (r.output) {
+      result = r as BuildResult;
```

Comment:
So this why returning a `version` would be good because then you wouldn't need to cast. Oh well 🤷‍♂️